modules/EventEmitter.js

```diff
@@ -488,10 +488,10 @@ class EventEmitter {
                     seller_erc_id,
                     price,
                 } = message;
-                // await dvController.handleNewDataSeller(
-                //     purchase_id, seller_erc_id, seller_node_id,
-                //     data_set_id, ot_object_id, price,
-                // );
+                await dvController.handleNewDataSeller(
+                    purchase_id, seller_erc_id, seller_node_id,
+                    data_set_id, ot_object_id, price,
+                );
```

Comment:
I'm pretty sure this was commented out due to the design spec stating that a buyer does not become a seller after they purchase data

Reply:
Agreed